<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particle Filter Toolbox</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/PFT.PNG</normaloff>:/icons/PFT.PNG</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#groupBox { 
	font: 87 8pt &quot;Arial Black&quot;;
	 color: rgb(114, 114, 114);
}


#groupBox_2 {
	font: 87 8pt &quot;Arial Black&quot;;
	 color: rgb(114, 114, 114);
 }

#groupBox_3 {
	font: 87 8pt &quot;Arial Black&quot;;
	 color: rgb(114, 114, 114);
 }

#groupBox_4 { 
	font: 87 8pt &quot;Arial Black&quot;;
	 color: rgb(114, 114, 114);
}

#lineEdit_signal {
	background-color: rgb(227, 227, 227);
}

#lineEdit_GT {
	background-color: rgb(227, 227, 227);
}

#spinBox_NoParticulas{
	background-color: rgb(227, 227, 227);
}

#spinBox_resample{
	background-color: rgb(227, 227, 227);
}

#spinBox_salida{
	background-color: rgb(227, 227, 227);
}

#spinBox_runs{
	background-color: rgb(227, 227, 227);
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>8</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Filter design</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Filter algorithm:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_filtro">
         <item>
          <property name="text">
           <string>SIS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SIR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>G_PF</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>APF</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Number of particles:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBox_NoParticulas">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5000</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_Tresample">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Resampling algorithm:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_Tresample">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Multinomial</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Systematic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Stratified</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Residual</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_resample">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Resampling percent:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinBox_resample">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Estimation algorithm:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBox_salida">
         <item>
          <property name="text">
           <string>Weighted mean</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Maximum weight</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Robust mean</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_salida">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Robust mean percent:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="spinBox_salida">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>25</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>8</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Model</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>State space:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBox_ee">
         <item>
          <property name="text">
           <string>[x, y, Vx, Vy] (dynamic bbox)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>[x, y, Vx, Vy, w]</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>[x, y, Vx, Vy, w, Vw]</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Observation model:    </string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_obsmodel">
         <item>
          <property name="text">
           <string>HSV color-based</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LBP-based</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Arial Black</family>
        <pointsize>8</pointsize>
        <weight>10</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="title">
       <string>Video</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Input video:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_signal">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_signal">
         <property name="text">
          <string>Examine ...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Annotations:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEdit_GT">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_gtSearch_2">
         <property name="text">
          <string>Examine ...</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
      <zorder></zorder>
      <zorder>layoutWidget</zorder>
      <zorder>label_7</zorder>
      <zorder>label_10</zorder>
      <zorder></zorder>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>17</width>
        <height>1</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Number of algorithm runs:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_runs">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_save">
        <property name="text">
         <string>Save estimation</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_process">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="PB_aceptar">
        <property name="text">
         <string>Run</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="PB_cancelar">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBox_runs</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>464</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menupf">
    <property name="title">
     <string/>
    </property>
   </widget>
   <addaction name="menupf"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
